### PR TITLE
fix(tools): fix insert-erms helper script

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Without fix, code with `html` and `css` would fail, as erm is not defined in one of them.

### CODE REVIEW

**Review by ojeytonwilliams — COMMENTED** [review]
Isn't this attacking the problem from the wrong direction?  Surely we want `insertErms` to fail (or at least warn) if you try and insert zero or one erm, since that's pointless or incorrect, respectively.

How about just leaving `insertErms` as is and only calling it if you have two markers to insert?

**Comment by ShaunSHamilton** [comment]
> How about just leaving `insertErms` as is and only calling it if you have two markers to insert?

Yes, this makes sense 👍

File: tools/challenge-helper-scripts/helpers/get-step-template.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ function getStepTemplate({
 }) {
   const seedTexts = Object.values(challengeSeeds)
     .map(({ contents, ext, editableRegionBoundaries }) => {
-      const fullContents = insertErms(contents, editableRegionBoundaries);
+      let fullContents = contents;
+      if (editableRegionBoundaries.length >= 2) {
```
Comment (ojeytonwilliams) [inline tools/challenge-helper-scripts/helpers/get-step-template.js:31]:
```suggestion
      if (editableRegionBoundaries.length === 2) {
```
Nitpicky, I know, but it currently does have to be exactly 2.

Comment (ShaunSHamilton) [reply tools/challenge-helper-scripts/helpers/get-step-template.js:31]: I thought the idea was to allow it, but ignore it: https://github.com/freeCodeCamp/freeCodeCamp/pull/42903/commits/e46fa3099dcb07c445e3b1ceb867b43556bf359a#diff-785b651b36e382407def34a0ca4d23fbad88db34d2611924ecc572047ce86b18R32

Comment (ojeytonwilliams) [reply tools/challenge-helper-scripts/helpers/get-step-template.js:31]: I think we want to know if the number of erms is wrong. It's not a big deal either way, so we can merge as is if you like.

**Review by ojeytonwilliams — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**